opt_einsum/parser.py

```diff
@@ -3,11 +3,61 @@
 """
 
 import numpy as np
+import sys
+
+einsum_symbols_base = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ'
```

Comment:
Can we build this string like below? `"''.join(map(chr, range(65, 91)))`

Reply:
One could write this as (to get both lower- and upper-case)
```python
''.join(map(chr, range(97, 123))) + ''.join(map(chr, range(65, 91)))
```
but I'm tempted to leave the current string as its shorter and a bit more obvious?